fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -262,10 +218,7 @@ public boolean isWritable(final Graph graph, final Node subject) {
      * @return whether the subject is frozen node
      */
     public boolean isFrozenNode(final Graph graph, final Node subject) {
-        final Iterator<Triple> objects = getObjects(graph, subject, RdfLexicon.HAS_PRIMARY_TYPE);
-        return objects.hasNext()
-                && objects.next().getObject()
-                .getLiteralValue().toString().equals("nt:frozenNode");
+        return getValue(graph, subject, HAS_PRIMARY_TYPE.asNode()).filter("nt:frozenNode"::equals).isPresent();
```

Comment:
I actually suspect that there is no test for that, since I wasn't getting test failures with #935 
If we merge this PR first, I can amend #935 to test this -- it will probably involve changing that function somewhat.